backend/tests/services/knowledge/group_kb/test_group_kb_crud.py

```
@@ -0,0 +1,598 @@
+            )
+
+
+class TestDeleteKnowledgeBase:
+    """Test knowledge base deletion permissions."""
+
+    def test_owner_can_delete_kb(
+        self,
+        test_db: Session,
+        test_group,
+        test_group_owner,
+        test_group_maintainer,
```

Ruff (0.15.2)

[warning] Unused method argument: `test_group_maintainer`

(ARG002)